test/cpp/end2end/client_lb_end2end_test.cc

```diff
@@ -239,6 +240,7 @@ class ClientLbEnd2endTest : public ::testing::Test {
     servers_.clear();
     creds_.reset();
     grpc_shutdown_blocking();
+    grpc_maybe_wait_for_async_shutdown();
```

Comment:
If this is the case, then we do not need to use _blocking version right above. I think it might be better to add a comment to avoid confusion.